tests/csapi/upload_keys_test.go

```diff
@@ -180,11 +180,11 @@ func TestKeyClaimOrdering(t *testing.T) {
 	deployment := complement.Deploy(t, 1)
 	defer deployment.Destroy(t)
 	alice := deployment.Register(t, "hs1", helpers.RegistrationOpts{})
-	_, oneTimeKeys := alice.MustGenerateOneTimeKeys(t, 2)
+	deviceKeys, oneTimeKeys := alice.MustGenerateOneTimeKeys(t, 2)
 
 	// first upload key 1, sleep a bit, then upload key 0.
 	otk1 := map[string]interface{}{"signed_curve25519:1": oneTimeKeys["signed_curve25519:1"]}
-	alice.MustUploadKeys(t, nil, otk1)
+	alice.MustUploadKeys(t, deviceKeys, otk1)
```

Comment:
This is the only place where we didn't do this. Are we sure this wasn't on purpose (should have had a comment if-so)?

This test is about one-time key claim ordering so it shouldn't matter ⏩